index.html

```
@@ -4059,7 +4043,58 @@ <h3>Appearance</h3>
+                </div>
+              </div>
+
+              <div class="settings-group">
+                <div class="settings-group-label">Numista search patterns</div>
+                <p class="settings-subtext">
+                  Add your own patterns to rewrite Numista search queries. Custom rules always
+                  apply. Override order: first match wins.
+                </p>
+                <div class="chip-grouping-controls">
+                  <input
+                    type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)